storage/storage.cpp

```diff
@@ -770,6 +771,11 @@ void Storage::RegisterCountryFiles(TIndex const & index, string const & director
   TLocalFilePtr localFile = GetLocalFile(index, version);
   if (localFile)
     return;
+  // @TODO(bykoianko) This place should be implemented faster.
+  // You use Find twice. Here and in GetCountryFile.
```

Comment:
I've rewritten the comment. I'm not in a hurry to optimize all such places but prefer to leave some marks where there's some place for optimization.